### PR TITLE
remove statistics module

### PR DESCRIPTION
**Pull request by alexweber**
including references in search.test and default.settings.php

### CODE REVIEW

File: core/modules/search/search.test [hunk 1]
```diff
@@ -437,14 +437,6 @@ class SearchRankingTestCase extends DrupalWebTestCase {
...
     $this->drupalPost(NULL, $edit, t('Save'));
 
-    // Enable counting of statistics.
-    variable_set('statistics_count_content_views', 1);
-
-    // Then View one of the nodes a bunch of times.
-    for ($i = 0; $i < 5; $i ++) {
-      $this->drupalGet('node/' . $nodes['views'][1]->nid);
-    }
-
```
Comment (sun) [inline core/modules/search/search.test:447]: I'm fairly sure you deleted the test setup/preparation, but the actual results are still expected later on in this test.

Comment (sun) [inline core/modules/search/search.test:447]: Oh, I might be wrong.  You also removed the `views` element from the list of ranks above.  Guess that works :)

Comment (alexweber) [inline core/modules/search/search.test:447]: Yup, so only the comment will affect the ranking now.  _Should_ be fine :)

**Comment by quicksketch** [comment]
Tested this out locally and it all looks great! I couldn't find any remaining references to statistics other than node_comment_statistics, which is of course different than statistics module itself. All tests pass, great job! Merged.

**Pull request merged**